ElectronClient/gui/NoteEditor/NoteBody/AceEditor/AceEditor.tsx

```diff
@@ -535,9 +535,15 @@ function AceEditor(props: NoteBodyEditorProps, ref: any) {
 
 	useEffect(() => {
 		if (props.searchMarkers !== previousSearchMarkers || renderedBody !== previousRenderedBody) {
-			webviewRef.current.wrappedInstance.send('setMarkers', props.searchMarkers.keywords, props.searchMarkers.options);
+			if (props.searchMarkers.keywords.length) {
+				// find-in-note highlighting
+				webviewRef.current.wrappedInstance.send('setMarkers', props.searchMarkers.keywords, props.searchMarkers.options);
+			} {
+				// search words highlighting (use props.highlightedWords instead of props.searchMarkers.keywords)
+				webviewRef.current.wrappedInstance.send('setMarkers', props.highlightedWords, props.searchMarkers.options);
```

Comment:
Any reason why there has to be two properties for the searchMarkers / highlightedWords for highlighting? Couldn't you update useSearchMarkers so that it gives the data you need?